### ISSUE TITLE
Make JSON lib configurable

### ISSUE THREAD

**Issue by abitdodgy** [smeevil/cloudex#57]
New versions of Phoenix and Ecto are using Jason as the default JSON library. Any interest in making the JSON library configurable? Something like...

```elixir
config :cloudex, :json_library, Jason
```

Then, whenever we are calling it, we can do something like...

```elixir
@json_library Application.get_evn(:cloudex, :json_library)

def foo do
  @json_library.decode!(...)
end
```

**Comment by smeevil** [comment]
Sounds like a good plan 👍 otherwise making Jason the default would be fine as well.

**Comment by abitdodgy** [comment]
@smeevil see #58

**Comment by abitdodgy** [comment]
Merged in #58
